index.html

```diff
@@ -1145,6 +1150,19 @@ <h3>Directory Service API</h3>
                     </ul>
                 </p>
 
+                <p>
+                  <span class="rfc2119-assertion" id="tdd-http-alternate-content">
+                     A Directory server MAY provide alternative representations through
+                     server-driven content negotiation, that is by honouring the 
+                     request's Accept header and responding with the supported
+                     TD serialization and equivalent Content-Type header.
+                  </span>
+                  <span class="rfc2119-assertion" id="tdd-http-alternate-language">
+                     A Directory server MAY provide modified TDs or error responses using a different
+                     default language after server-driven content negotiation, 
+                     that is by honouring the request's Accept-Language header.
+                  </span>
```

Comment:
We have an assertion bellow to allow directories to both receive and respond different content types. Should the directory also accept different languages from the client (when Content-Language is set on POST/PUT/PATCH).

https://github.com/w3c/wot-discovery/pull/318/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R1210

Should these assertions be for the whole directory incl. Search and notification or just Things?